modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/core/useIE11Compatibility.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {useLayoutEffect} from 'react';
+
+export default function useIE11Compatibility() {
```

Comment:
I implemented this a hook to have a cleanup method where restore the original function

Reply:
I believe that ie11 doesn't support SPA, it seems to be doing a full page reload after any interaction, so this shouldn't be needed?

Reply:
> I believe that ie11 doesn't support SPA

I don't recall for sure; I _thought_ that it did, but i could be mistaken. @julien, @jbalsas, do you remember?

Reply:
I also thought IE 11 supported SPA

Reply:
Maybe is disabled by default then? 🤔

Reply:
IE11 supports SPA. Options were added to disable it in https://github.com/liferay/liferay-portal/commit/317dc5429e0ff6ac8682d9e9f843558744fb6097 as part of [Provide a configuration option to disable SPA in IE11](https://issues.liferay.com/browse/LPS-102605)

Reply:
Supportability decided to ship it disabled by default back then if I remember correctly

Reply:
Ok so it makes sense to keep the cleanup just in case, thanks!